### ISSUE TITLE
Documentation description by Area

### ISSUE THREAD

**Issue by charliefancelli** [nelmio/NelmioApiDocBundle#1307]
Hi,

We try to show a different documentation description to each Area but actually i think it's not possible.

Do you think it will be possible to define something like this ?

```
nelmio_api_doc:
    areas:
        default:
            path_patterns: [ ^/api ]
            host_patterns: [ ^api\. ]
        internal:
            path_patterns: [ ^/internal ]
            documentation:
                description: "This is the documentation for internal"
        commercial:
            path_patterns: [ ^/commercial ]
            documentation:
                description: "This is the documentation for commercial"
```

or like this 

```
nelmio_api_doc:
    areas:
        default:
            path_patterns: [ ^/api ]
        internal:
            path_patterns: [ ^/internal ]
        commercial:
            path_patterns: [ ^/commercial ]

    documentation:
         default:  
             info:    
                  title: Internal Documentation
                  version: "1.1"
                 description: "This is the documentation for internal"

        commercial:  
             info:   
                  title: Commercial Documentation
                  version: "1.0" 
                 description: "This is the documentation for commercial"
```

The second solution could be great to specify all the configuration.

**Comment by GuilhemN** [comment]
I think I prefer your first proposal since it allows to have a common base between areas.
PR welcome if you're willing to do it :) Adding an external describer for each area should be enough.

**Comment by GuilhemN** [comment]
Fixed in https://github.com/nelmio/NelmioApiDocBundle/pull/1404!

**Comment by andydandy80** [comment]
You wrote it was fixed in #1404, but in current version its not working, why removed code? Whats the correct way to describe areas?
